scenarios/Azlaroc/sftpgo-bf.yaml

```diff
@@ -0,0 +1,17 @@
+type: leaky
+name: Azlaroc/sftpgo-bf
+description: "Detect SFTPGo bruteforce attacks on FTP/SSH"
+filter: "evt.Meta.log_type == 'sftpgo_auth' && evt.Meta.is_failed_login == 'true'"
+groupby: evt.Meta.source_ip
+capacity: 3
+leakspeed: "30s"
+blackhole: 4h
```

Comment:
make this like `5m` instead `blackhole` doesnt control the ban duration just the supression on repeated buckets if enforce happens at later time.